### PR TITLE
Search-bar-created

### PR DESCRIPTION
**Pull request by molloyconor511**

### CODE REVIEW

File: src/client/App.css [hunk 1]
```diff
@@ -39,4 +39,53 @@
...
+
+.search-bar {
+  margin-bottom: 40px;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.myButton {
```
Comment (vherus) [inline src/client/App.css:52]: It's best to pick one coding style - either use camelCase, or kebab-case, but try not to mix and match.

File: src/client/Components/Calendar.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export default function Results() {
```
Comment (vherus) [inline src/client/Components/Calendar.jsx:1]: Keep the file names the same as the component name to make it easier to navigate the project, this function should probably be called `Calendar`

File: src/client/Components/Search.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+//import e from "express";
+import { useState } from "react";
+
+const SearchByState = () => {
```
Comment (vherus) [inline src/client/Components/Search.jsx:4]: Same comment RE naming files & components

**Review by vherus — APPROVED** [review]

File: src/client/App.css (hunk 1 above)
Comment (molloyconor511) [reply src/client/App.css:52]: Agreed on camelCase- will change all existing code

**Review by molloyconor511 — COMMENTED** [review]
Issues with different styles due to switching repo after 3 days with mix between baseline styling and newly implemented; allows should have more detailed discussions with other team members regarding same

**Pull request merged**